Detectors/MUON/MCH/Simulation/CMakeLists.txt

```diff
@@ -3,27 +3,33 @@ set(MODULE_NAME "MCHSimulation")
 O2_SETUP(NAME ${MODULE_NAME})
 
 set(SRCS
-   src/Detector.cxx
-   src/Geometry.cxx
-   src/GeometryTest.cxx
-   src/Hit.cxx
-   src/Materials.cxx
-   src/Materials.h
-   src/Station1Geometry.cxx
-   src/Station1Geometry.h
-   src/Station2Geometry.cxx
-   src/Station2Geometry.h
-   src/Station345Geometry.cxx
-   src/Station345Geometry.h
-   src/Stepper.cxx
-   src/Stepper.h
+  src/Detector.cxx
+  src/Geometry.cxx
+  src/GeometryTest.cxx
+  src/Hit.cxx
+  src/Materials.cxx
+  src/Materials.h
+  src/Station1Geometry.cxx
+  src/Station1Geometry.h
+  src/Station2Geometry.cxx
+  src/Station2Geometry.h
+  src/Station345Geometry.cxx
+  src/Station345Geometry.h
+  src/Stepper.cxx
+  src/Stepper.h
+  src/Digit.cxx
```

Comment:
this is nitpicking I admit, but I would prefer to keep alphabetical order in the lists

Reply:
Ok, done in commit.